content/en/real_user_monitoring/retention_analysis/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Retention Analysis
+kind: documentation
+---
+
+## Overview
+Retention Analysis allows you to measure how often users are successfully returning to a page or action. By tracking user retention over time, you can gain insights around overall user satisfaction.
+
+A cohort is a group of users who participate in an initial event, such as clicking a link. A user in the cohort is considered retained if they subsequently complete a return event, such as clicking the same link again or clicking a **Proceed to Payment** button. Only views and actions can act as events.
```

Comment:
```suggestion
A cohort is a group of users who participate in an initial event, such as clicking a link. A user in the cohort is considered retained if they subsequently complete a return event, such as clicking the same link again or clicking a **Proceed to Payment** button. Only views and actions can act as events.
```

This is the first time we are reading about a cohort and in this context I think it would flow better to re-word this a bit, I initially felt maybe I missed a step or concept when I started reading this. Something that would match the action being depicted in the screenshot, possibly:

View and define the _retention cohort_ to help track user retention. A cohort is a group of users who participate in an initial event, such as clicking a link. A user in the cohort is considered retained if they subsequently complete a return event, such as clicking the same link again or clicking a Proceed to Payment button. Only views and actions can act as events.